src/Swashbuckle.AspNetCore.SwaggerUI/SwaggerUIOptionsExtensions.cs

```diff
@@ -333,5 +333,10 @@ public static void UseResponseInterceptor(this SwaggerUIOptions options, string
         {
             options.Interceptors.ResponseInterceptorFunction = value;
         }
+
+        public static void EnableSwaggerDocumentUrlsRoute(this SwaggerUIOptions options)
```

Comment:
I wonder if we should call it an "endpoint" rather than a route?

Reply:
Fixed